### DISCUSSION TITLE
problems with LBB and touchscreen

### DISCUSSION THREAD

**Discussion by PPrecht** [outdoorbits/little-backup-box#326] [Q&A]
Hello,

I've been running an LBB with a small screen for some time now.

Now I wanted an LBB with a touchscreen and bought this one.   [https://amzn.eu/d/iFeTr6a](url)

So I installed Pi OS Desktop with the imager and then made the entries in the config.txt according to the manufacturer's instructions.

When I start now, the comitup WLAN mask appears, which I can't get past without selecting a WLAN - but I don't want to join a WLAN but want to be able to access the LBB with my smartphone when needed.

![IMG_8678](https://github.com/user-attachments/assets/32e2b7e1-861c-4769-bff7-ae288dd82436)

When LBB starts, it works without any problems - I just don't want to mess around with a mouse and keyboard when I'm on the go.

Is there an instruction how I can get the touchscreen to work without a question to choose WLAN and starting LBB?

i tried to edit the autostart für LBB, but the WLAN mask seems to index.php, when i set it to system.php it also opens the WLAN mask

What i am making wrong?

Thanks
Peter

**Comment by PPrecht** [comment]
I have now continued testing: If the white screen appears and nothing happens, all you have to do is press the raspberry key on the keyboard once so that LBB goes to the foreground.

https://youtu.be/JdsBVf9ubDk

If the network selected when setting up Comitup is not available, this selection screen appears - how do I get over it - or: how long do I have to wait for it to disappear on its own?

https://youtu.be/I08FGh-oeD8

In some cases this warning also appears, for which you have to create an exception in Firefox

https://youtu.be/HegrqixcTr0

**Comment by PPrecht** [comment]
Hello,

today i installed Pi OS in 32 bit.

After installing LBB with comitup - same procedure.

Comitup login screen appears - only when i connect the box to my routers WLAN. it will go further.
When the white screen appears i have to hit the Raspberry key on the keyboard to see the LBB UI.

When comitup connects to the routers WLAN, i have no chance to connect my smartphone to LBB

When i deactivate my routers WLAN the login screen from Comitup will appear

![comitup](https://github.com/user-attachments/assets/e0e3e407-dbd3-428a-a234-d437ecabd157)

@outdoorbits 

Is there  possibility to make  timers? perhabs make a checkbox in LBBs settings

1. if the white page is showing for 10 seconds hit the rasbperry key per code
2. if in comitup no WLAN will connect for 10 seconds switch to the LBBs interface

Falls es Dir helfen würde, könnte ich dir meine Box gerne per Post zusenden - damit dürfte die Ursachenforschung für Dich einfacher sein.

Thanks

Peter

**Comment by dietrest** [comment]
I get this screen if I access https://10.41.0.1 to get lbb I have to use http://10.41.0.1:8080
